src/test/java/unit/GcpNotebookControlled.java

```diff
@@ -39,7 +39,12 @@ void listDescribeReflectCreateDelete() throws IOException {
     String name = "listDescribeReflectCreateDelete";
     UFGcpNotebook createdNotebook =
         TestCommand.runAndParseCommandExpectSuccess(
-            UFGcpNotebook.class, "resource", "create", "gcp-notebook", "--name=" + name);
+            UFGcpNotebook.class,
+            "resource",
+            "create",
+            "gcp-notebook",
+            "--name=" + name,
+            "--metadata=foo=bar");
 
     // check that the name and notebook name match
     assertEquals(name, createdNotebook.name, "create output matches name");
```

Comment:
do we need to add the assertion for the metadata?

Reply:
The metadata doesn't show up when we describe the resource; it's cloud-only. If we had a convenient way to get it from the cloud, we could assert on that.